web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -28,8 +28,12 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['#title'] = $this->t('GFW');
 
     $form['placeholder'] = [
```

Comment:
Let's rename this 'placeholder' for something more meaningful

Reply:
@pcambra  have renamed it to `gfw_map_placeholder` will that do, or we can have something more better for this ?